rubocop.yml

```diff
@@ -59,6 +59,8 @@ Metrics/MethodLength:
 
 Metrics/ParameterLists:
   Max: 4
+  Exclude:
+    - "**/*_spec.rb"
```

Comment:
hmm, what about refactoring that spec helper method to accept an `options` hash instead? Then you can have as many parameters as you want, and the keys describe their semantics. 

In this particular case I happen to side with the cop; I would even say when it comes to positional arguments, the limit four is not hard enough: when you’re not looking at a method with four arguments, you’re almost guaranteed to forget which is which

Reply:
It's actually an explicit choice I made to not use an option object.

As an example, please check the current file in loveos-backend and especially the `atttributes_expectations` method: spec/api/containers/checkouts/seller_calculation_spec.rb

It allows me to test my calculation service in one line by just changing numbers in the params. Using an option object would not provide me any benefit from using the `attributes_expectation` method

Reply:
hmm, I guess I see your point… that method looks like it wants to be a `shared_examples` block, but that, too, would be more verbose